ocetrac/model.py

```diff
@@ -44,8 +47,16 @@ def __init__(self, da, mask, radius, min_size_quartile, xdim, ydim):
         self.mask = mask
         self.radius = radius
         self.min_size_quartile = min_size_quartile
+        self.timedim = timedim
         self.xdim = xdim
         self.ydim = ydim   
+        
+        if ((timedim, ydim, xdim) != da.dims):
+            try:
+                da = da.transpose(timedim, ydim, xdim) 
+            except:
+                raise ValueError(f'Ocetrac currently only supports 3D DataArrays. The dimensions should only contain ({timedim}, {xdim}, and {ydim}). Found {list(ds.dims)}')
```

Comment:
Ugh sorry I committed directly, so feel free to revert. This basically just uses the actual names given for `timedim` etc.